backend/post-recommendations/post-recommendations.py

```diff
@@ -0,0 +1,94 @@
+# %%
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+LONG_POST_UPPER_LIMIT = 2000   ## Ideal upper length (characters) for longer posts
+LONG_POST_LOWER_LIMIT = 1300    ## Ideal lower length (characters) for longer posts
+SHORT_POST_UPPER_LIMIT = 300    ## Ideal upper length (characters) for shorter posts
+SHORT_POST_LOWER_LIMIT = 150    ## Ideal lower length (characters) for shorter posts 
+
+plot_points = {}
+x_vals = []
+y_vals = []
+
+
+def wordIntentValueTransformation(x):
+    """Transforms a value between 0 and 100 to a value between 0 to 1.
+    Produces a value used to assess the intent of a word.
+    Uses the function y = -(1/(x + 1)) + 1."""
+
+    return (-1 * (1 / (float(x) + 1))) + 1
+
+
+def getDistance(point_one, point_two):
+    # TODO
+
+    return
+
+
+def postToPoint(postInfo):
+    """Converts post information to a plot point, (x, y).
+    Factors in things like post engagement and post quality.
+    Also used to replot posts when some interactions with post occur."""
+
+    ## Calculating x-value for the post.
+    ## Is calculated based on interactons with the post, such as the number of "comments" for the posts, the average length of these "comments", how many shares a post has, and information about the post's author.
+    x_val = 0
+
+    # TODO
+
+    ## Calculating y-value for the post.
+    ## Is calculated based on post content
+    y_val = 0
+
+    ## Ratings ranging from 0 to 1.
+    postTextLength = len(post["text"])
+    strippedpostText = post["text"].replace(" ", "")
+    
+    questionMarkCount = post["text"].count("?")
+    periodCount = post["text"].count(".")
+    exclamationPointCount = post["text"].count("!")
+
```

Comment:
so in Python, we actually don't use `camelCasing` -> we rather use `snake_casing`